OrangeDesignSystem/Sources/OrangeDesignSystem/Modules/About/Internal/LitsItems/AppNews/AppNewsViews.swift

```diff
@@ -34,31 +34,59 @@ struct AppNewsList: View {
 
     var body: some View {
         ScrollView {
-            if releaseDescriptions.isEmpty {
-                Text(°°"modules.about.app_news.no_news")
-            } else {
-                ForEach(releaseDescriptions, id: \.version) { releaseDescription in
-                    AboutReleaaseDescriptionEntry(releaseDescription: releaseDescription)
-                        .padding(.horizontal, ODSSpacing.m)
-                    Divider()
-                }
+            switch viewModel.releaseDescriptions {
+            case .loading:
+                loadingView()
+            case let .loaded(releaseDescription) where !releaseDescription.isEmpty:
+                loadedView(releaseDescription)
+            default: // case .error, case .loaded(let releaseDescription) where releaseDescription.isEmpty:
+                errorView()
             }
+        }.onAppear {
+            viewModel.load()
         }
     }
+
+    // ===========
+    // MARK: Views
+    // ===========
+
+    private func loadingView() -> some View {
+        Text(°°"shared.loading")
```

Comment:
not clear for me the °°

Reply:
Found it funny, but I will use another property if you prefer :D